resteasy-core-spi/src/main/java/org/jboss/resteasy/spi/BuiltResponseProcessor.java

```diff
@@ -0,0 +1,13 @@
+package org.jboss.resteasy.spi;
+
+import org.jboss.resteasy.spi.metadata.ResourceMethod;
+
+import javax.ws.rs.core.Response;
+
+public interface BuiltResponseProcessor {
```

Comment:
Same here, I think this can be implemented as a combination of DynamicFeature and ContainerFilterResponse.  You scan annotatoins and view return types within DynamicFeature.  You add a ContainerFilterResponse for the method if its a Spring one.  This filter handles the transformation between the spring return object and jaxrs Response.

Reply:
Thanks for the tip, looking into it